packages/fxa-settings/src/components/UnitRow/index.tsx

```diff
@@ -138,12 +139,16 @@ export const UnitRow = ({
 
   return (
     <div className="unit-row">
-      <div className="unit-row-header">
+      <div className="font-header w-full mb-1 mobileLandscape:flex-none mobileLandscape:mb-0 mobileLandscape:mr-2 mobileLandscape:w-40">
```

Comment:
The `.unit-row-header h3` custom style bothered me since it doesn't adhere to TW best practices so I at first created a `UnitRowHeader` component and then noticed a slight DOM element difference, and then realized we only need these classes in two spots anyway and I don't foresee us needing another `UnitRow` component anytime in the near future, so I removed those custom styles and added those classes on directly.